### PR TITLE
Add Misc Quest Scripts

### PR DESCRIPTION
**Pull request by NickTyrer**
## 🍰 Pullrequest

### CODE REVIEW

**Comment by ratkosrb** [comment]
Oh damn, didn't see you made a pull request with it. Your version is different though. What did you base it on? There were no emotes or flag changing in the sniff I used, just spell cast and the text.

**Comment by Rushor** [comment]
@NickTyrer I really love you waypoint_data work. If you wanna work with me we can handle sth out, please contact me with PBienati@web.de if you wish :)

File: sql/migrations/20211203202918_world.sql [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+IF v=0 THEN
+INSERT INTO `migrations` VALUES ('20211203202918');
+-- Add your query below.
+
+
+-- Add Scripts to Quest Stones of Binding (credit cmangos)
+INSERT INTO `gameobject_scripts` (`id`, `delay`, `command`, `datalong`, `datalong2`, `datalong3`, `datalong4`, `target_param1`, `target_param2`, `target_type`, `data_flags`, `dataint`, `dataint2`, `dataint3`, `dataint4`, `x`, `y`, `z`, `o`, `condition_id`, `comments`) VALUES
+(15216, 0, 9, 443, 62, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 'Stones of Binding : Stone of East Binding - Respawn Gameobject'),
+(16598, 0, 9, 444, 62, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 'Stones of Binding : Stone of Outer Binding - Respawn Gameobject'),
+(15398, 0, 9, 445, 62, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 'Stones of Binding : Stone of West Binding - Respawn Gameobject');
```
Comment (ratkosrb) [inline sql/migrations/20211203202918_world.sql:16]: I am confused whats going on here. Why are you respawning Incendicite Mineral Veins upon using this object?

File: sql/migrations/20211203202918_world.sql [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+INSERT INTO `quest_end_scripts` (`id`, `delay`, `command`, `datalong`, `datalong2`, `datalong3`, `datalong4`, `target_param1`, `target_param2`, `target_type`, `data_flags`, `dataint`, `dataint2`, `dataint3`, `dataint4`, `x`, `y`, `z`, `o`, `condition_id`, `comments`) VALUES
+(651, 0, 9, 446, 62, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 'Stones of Binding : Stone of Inner Binding - Respawn Gameobject');
+UPDATE `quest_template` SET `CompleteScript` = 651 WHERE `entry` = 651;
+
+INSERT INTO `gameobject` (`guid`, `id`, `position_x`, `position_y`, `position_z`, `orientation`, `spawntimesecsmin`, `spawntimesecsmax`) VALUES 
+(443, 2870, -846.38556, -3268.444, 78.038246, 2.8099754, -62, -62),
+(444, 2870, -1351.0742,-2740.2263, 59.40781, 4.520403, -62, -62),
+(445, 2870, -855.4827, -1783.2532, 39.629177, 4.3109655, -62, -62),
+(446, 2870, -1510.17, -2171.2039, 17.2624, 4.34587, -62, -62);
```
Comment (ratkosrb) [inline sql/migrations/20211203202918_world.sql]: Oh you are adding new gameobjects, but those guids are taken.

**Pull request merged**